iceoryx_examples/request_response/README.md

```diff
@@ -0,0 +1,214 @@
+# request_response
+
+## Introduction
+
+This example demonstrates how to use iceoryx in a client-server architecture
+using the request-response communication pattern.
+
```

Comment:
Could you please describe shortly the setup. One client sends a request of two fibonacci numbers and the servers sums them up and response with the third number.

It is helpful to have a rough idea what is being implemented here.